cluster/images/etcd-version-monitor/etcd-version-monitor.go

```diff
@@ -0,0 +1,275 @@
+/*
```

Comment:
Couldn't comment above, so commenting here.

Where those binaries come from? It's not a good practise to submit binaries to the repository.

Instead of that, you should download them as part of building the image.